frontend/src/components/pages/CreateAccountPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Text, Button, Box } from "@chakra-ui/react";
+import { ReactComponent as Logo } from "../../assets/Sistering_Logo.svg";
+
+const CreateAccountPage = (): React.ReactElement => {
+  const onContinue = () => {
+    //
+  };
+
+  return (
+    <Box
+      style={{
+        width: "100%",
+        display: "flex",
+        flexDirection: "row",
+        height: "100vh",
+      }}
+    >
```

Comment:
Almost there! Can you make use of the built-in chakra UI props? It'll make things a lot cleaner :P 
```suggestion
    <Box
      width="100%"
      display="flex"
      flexDirection="row"
      height="100vh"
    />
```

Same thing for below